### PR TITLE
St2/st3 after restructure

### PR DESCRIPTION
**Pull request by pahlers**
Let me tell you first it's a pretty long time I programmed Python. Probably someone with more Python skills would do this better.

I'm made it work for st2 and st3. It's need some testing. But I think it's all oké. Please test for both platforms. 

When I'm testing. I'm use to restart SublimeText. Otherwise you got caching problems. It's not the best way to test your code, and I think there must be a better way.

### CODE REVIEW

**Comment by benschwarz** [comment]
_WHOA_ - Great work @eonlepapillon, I'll review, test and look at getting this merged asap.

File: bower/utils/api.py [hunk 1]
```diff
@@ -1,40 +1,41 @@
...
-    try:
-      return json.loads(responseText)
-    except:
-      sublime.error_message('Oh Snap! It looks like theres an error with the Bower API.')
-  
\ No newline at end of file
+        # try:
+        return json.loads(responseText.decode())
+        # except:
+            # sublime.error_message('Oh Snap! It looks like theres an error with the Bower API.')
```
Comment (benschwarz) [inline bower/utils/api.py:41]: Just wondering why you commented this out? Accidental?

**Comment by benschwarz** [comment]
Merged to master.

File: bower/utils/api.py (hunk 1 above)
Comment (pahlers) [inline bower/utils/api.py:41]: Not accidental. I didn't figure out the proper exception for this try-except statement.
Using only except is way to heavy. You block all exceptions, even when you can or want to solve them better.

If you know the exception, put the name in between except and the colon like I did on row 28.

**Pull request closed without merging**